JsonLD.php

```diff
@@ -171,6 +172,80 @@ public static function expand($input, $options = null)
         return $input;
     }
 
+
```

Comment:
All the changes in this file are completely unrelated to the PR. Please revert them.

Reply:
Right, sorry I mixed up several PRs together.